<?xml version="1.0"?>
<launch>
	<arg name="gui" default="False" />
	<param name="human_description" textfile="$(find human_moveit_config)/urdf/human.urdf" /> 
	<param name="use_gui" value="$(arg gui)"/>
	
	<node pkg="tf" type="static_transform_publisher" name="human_broadcaster" args="0. 0. 0. 0. 0. 0. 1. /base /human/base 100" />

	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
		<remap from="robot_description" to="human_description" />
		<remap from="joint_states" to="/human/joint_states" />	
	</node>
	<node name="human_state_publisher" pkg="robot_state_publisher" type="state_publisher" >
		<remap from="robot_description" to="human_description" />
		<remap from="joint_states" to="/human/inter_joint_states" />
	</node>
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find human_moveit_config)/config/display_urdf.rviz" required="true" />
</launch>
